src/App.css

```diff
@@ -43,6 +43,25 @@ a {
   text-decoration: none;
 }
 
+input,
```

Comment:
these kind of selectors are scary, since they won't get hashed by CSS Modules, and affect so many elements regardless of page or component. I hope you intend for these to be site-wide!

Reply:
I do! These are resets but I think you're right. This could be done with much more intent. Importing normalize.css and tailwind might be a better option than the route I've been going. I also just found out we don't have to match the example app so I might redesign the visuals for the app 🎉